modules/UI/UI.js

```diff
@@ -135,14 +136,16 @@ UI.unbindEvents = () => {
  * @param {string} name etherpad id
  */
 UI.initEtherpad = name => {
-    if (etherpadManager || !config.etherpad_base || !name) {
+    const etherpadBaseUrl = sanitizeUrl(config.etherpad_base);
```

Comment:
check for null/undefined either for `etherpad_base` or for the `url` function argument of `sanitizeUrl`

Reply:
Done.